src/app/client.ts

```diff
@@ -349,6 +349,10 @@ export class StreetManagerApiClient {
     return this.httpHandler<SampleInspectionTargetCreateResponse>(() => this.axios.post('/sample-inspection-targets', createSampleInspectionTargetRequest, this.generateRequestConfig(requestConfig)))
   }
 
+  public async generateSampleInspection(requestConfig: RequestConfig): Promise<void> {
+    return this.httpHandler<void>(() => this.axios.post('/sample-inspection', null, this.generateRequestConfig(requestConfig)))
```

Comment:
```suggestion
    return this.httpHandler<void>(() => this.axios.post('/sample-inspection',this.generateRequestConfig(requestConfig)))
```
Don't think we need the null here

Reply:
if we don't have the null here we'd be passing the request config to the data param? Maybe post is the wrong method? (Originally we had a body for delegated user fields but the audit event for this - which will be part of the job - doesn't require them)